### ISSUE TITLE
Connector is losing records

### ISSUE THREAD

**Issue by alexnu** [confluentinc/kafka-connect-hdfs#290]
We just completed configuration and deployment of two Kafka Connect workers with HDFS connectors and we left them running side by side with our current consumer tool (Gobblin) for 3 days. When comparing the records fetched by the two tools it seems that Kafka Connect is missing some records. It does not happen all the time but twice a day for almost 2-3 minutes connectors seem to not be writing any records on HDFS.

After inspecting the logs at the time of losses we found the following:

```

**Comment by ashikaumanga** [comment]
@alexnu did you find a solution for this?

**Comment by alexnu** [comment]
Nope, we just accepted the fact we could be losing a small amount of records every once in a while for various reasons and we don't use kafka connect as our primary ingestion.

@ashikaumanga are you facing the same issue?

**Comment by ashikaumanga** [comment]
@alexnu  yes, asked a question here :  https://stackoverflow.com/questions/60199734/hdfs-sink-connector-messages-are-dropped-while-written-to-hdfs?noredirect=1#comment106481979_60199734
